src/components/content/order/common/RegionSelection.tsx

```diff
@@ -39,15 +40,22 @@ export const RegionSelection = ({
                 <Space wrap>
                     <Select
                         className={serviceOrderStyles.selectBoxClass}
-                        defaultValue={selectRegion}
-                        value={selectRegion}
+                        defaultValue={selectRegion.name + '(' + selectRegion.site + ')'}
```

Comment:
space missing between name and brackets.

Reply:
Updated